File: Trimestre 8/BEJI/src/main/kotlin/com/example/BEJI/Proveedor/Proveedor.kt
package com.example.BEJI.Proveedor

class Proveedor(
    private var id : Int,
    private var nombre:String,
    private var nit :String,
    private var celular: String,
    private var email: String,
    private var direccion: String
){
    //getters
    fun getId(): Int{return  this.id}
    fun getNombre(): String{return this.nombre}
    fun getNit(): String{return this.nit}
    fun getCelular(): String{return this.celular}
    fun getEmail(): String{return this.email}
    fun getDireccion(): String{return this.direccion}
}

File: Backend/app/Models/Proveedor.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Model;
use Illuminate\Database\Eloquent\Factories\HasFactory;

class Proveedor extends Model
{
    use HasFactory;
    
    protected $table= 'proveedor';
    protected $fillable=[
        'nombre',
        'nit',
        'celular',
        'email',
        'direccion'
    ];
 
}
